prebuilts/rss3-node.toml

```diff
@@ -1,9 +1,9 @@
 #:schema ./schema.json
 
 id = "rss3"
```

Comment:
Would it better to mark this as "rss3-alpha"?

Reply:
yes